src/SFA.DAS.EAS.Portal.Client/Models/Concrete/AccountDto.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Types;
+
+namespace SFA.DAS.EAS.Portal.Client.Models.Concrete
+{
+    public class AccountDto : IAccountDto
+    {
+        public AccountDto(IEnumerable<AccountLegalEntityDto> accountLegalEntities)
+        {
+            AccountLegalEntities = accountLegalEntities;
+        }
+
+        [JsonProperty("accountId")]
+        public virtual long AccountId { get; private set; }
+
+        //todo: return List in dto's, not enumerable (so consumer can add to them without copying - return most specific type)
+        // https://stackoverflow.com/questions/15490633/why-cant-i-use-a-compatible-concrete-type-when-implementing-an-interface
+        [JsonProperty("accountLegalEntities")]
+        public virtual IEnumerable<IAccountLegalEntityDto> AccountLegalEntities { get; private set; }
+
+        [JsonProperty("organisations")]
+        public virtual IEnumerable<Organisation> Organisations { get; private set; }
```

Comment:
Why have we added Organisations when we already have AccountLegalEntities? They are the same thing.

Reply:
AccountLegalEntities should be removed and replaced with Organisations.  That is the business terminology.

Reply:
`AccountLegalEntity` is what they are called in the reservations system which publishes the event we consume,  [see here](https://github.com/SkillsFundingAgency/das-reservations/blob/master/src/SFA.DAS.Reservations.Domain/Employers/AccountLegalEntity.cs). It's also what they're called in [EAS](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/f67d8253551995b02b98387d8d76f2da72e0d6fc/src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/Account/AccountLegalEntity.cs), and throughout DAS ([example](https://github.com/SkillsFundingAgency/das-provider-relationships/blob/17af94a1922ae42adad071caae911adba8268498/src/SFA.DAS.ProviderRelationships/Models/AccountLegalEntity.cs)).

I believe it's important to try to be consistent with current usage, otherwise it'll cause confusion, especially as the domain contains both an [LegalEntity](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/f67d8253551995b02b98387d8d76f2da72e0d6fc/src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/Account/LegalEntity.cs) and an [AccountLegalEntity](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/f67d8253551995b02b98387d8d76f2da72e0d6fc/src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/Account/AccountLegalEntity.cs). If we change to calling it Organisation here, developers will not know if it relates to an `AccountLegalEntity`, `LegalEntity` or something new entirely.

The [event ](https://github.com/SkillsFundingAgency/das-reservations-api/blob/a9d4e09a38ade26f2325d288155e6c406ea2f2eb/src/SFA.DAS.Reservations.Messages/ReservationCreatedEvent.cs) we consume contains an AccountLegalEntityId and an AccountLegalEntityName, and I think it best to continue with the same property names and matching entity name.

Unfortunately, we don't have a DAS ubiquitous language dictionary, but we do have precedent, and I think we should stick with what it is currently called throughout DAS.

Reply:
We've gone with `Organisation` as the entity name, but storing the id as `AccountLegalEntityId`, so there is no confusion.